From: Yaroslav Halchenko <debian@onerussian.com>
Subject: BF: adding trailing } for Help in antsRegistrationSpaceTime.sh
  bug was introduced in 51b08bb5ab03275df81f6c7047ab7d2d59705c8a

--- a/Scripts/antsRegistrationSpaceTime.sh
+++ b/Scripts/antsRegistrationSpaceTime.sh
@@ -163,7 +163,7 @@ script by Nick Tustison
 
 HELP
     exit 1
-
+}
 
 function reportMappingParameters {
     cat <<REPORTMAPPINGPARAMETERS
