diff --git a/CMake/ANTSExternalData.cmake b/CMake/ANTSExternalData.cmake
index 437c5d5..3a0af63 100644
--- a/CMake/ANTSExternalData.cmake
+++ b/CMake/ANTSExternalData.cmake
@@ -31,14 +31,7 @@ file:///var/bigharddrive/%(algo)/%(hash)")
 mark_as_advanced(ExternalData_URL_TEMPLATES)
 list(APPEND ExternalData_URL_TEMPLATES
   # Local data store populated by the ITK pre-commit hook
-  "file:///${${PROJECT_NAME}_SOURCE_DIR}/.ExternalData/%(algo)/%(hash)"
-  # Data published by Iowa Psychiatry web interface
-  ## The primary home for data
-  "http://slicer.kitware.com/midas3/api/rest?method=midas.bitstream.download&checksum=%(hash)"
-  # Data published by MIDAS
-  "http://midas3.kitware.com/midas/api/rest?method=midas.bitstream.download&checksum=%(hash)&algorithm=%(algo)"
-  # Data published by developers using git-gerrit-push.
-  "http://www.itk.org/files/ExternalData/%(algo)/%(hash)"
+  "file:///${${PROJECT_NAME}_SOURCE_DIR}/debian/testdata/%(algo)/%(hash)"
   )
 
 # Tell ExternalData commands to transform raw files to content links.
diff --git a/debian/patches/series b/debian/patches/series
index e7918eb..5428a25 100644
--- a/debian/patches/series
+++ b/debian/patches/series
@@ -1 +1,2 @@
 up_use_bash
+add-soname.patch
